tyk-pro/templates/deployment-dash.yaml

```diff
@@ -25,6 +25,11 @@ spec:
       labels:
         app: dashboard-{{ include "tyk-pro.fullname" . }}
         release: {{ .Release.Name }}
+      {{- if .Values.enableIstioIngress }}
+      annotations:
+        traffic.sidecar.istio.io/excludeInboundPorts: "{{ .Values.dash.containerPort }}"
+        traffic.sidecar.istio.io/includeInboundPorts: ""
```

Comment:
Please see comment in `deployment-gw-repset.yaml` file regarding `InboundPorts`. Same question applies here as well.